### PR TITLE
#4400 [Manager] Remove NewWorkspaceContext

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4400)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/store/reducers/ui/intro.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  opened: false,
+};
+
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+    case actions.openIntro.TRIGGER:
+      return {
+        ...state,
+        isOpen: true,
+        opened: state.opened || true,
```
Comment (konekoya) [inline ohara-manager/client/src/store/reducers/ui/intro.js]: I'm a bit confused with this piece of state. Could you elaborate why we need this :)

Comment (eechih) [reply ohara-manager/client/src/store/reducers/ui/intro.js]: 我想表達 "只能被修改一次" 的意味～ :)

Comment (eechih) [reply ohara-manager/client/src/store/reducers/ui/intro.js]: 不過，這好像容易產生疑惑 :(

Comment (eechih) [reply ohara-manager/client/src/store/reducers/ui/intro.js]: @konekoya  Good catch, I will adjust to `opened: true`, thanks.

Comment (konekoya) [reply ohara-manager/client/src/store/reducers/ui/intro.js]: Ah, I see. So the intro dialog will only show up once, right?

Comment (eechih) [reply ohara-manager/client/src/store/reducers/ui/intro.js]: 正確的說，當使用者第一次使用 Ohara 時，用 opened 來判斷是否主動顯示 Intro dialog

```
export const useWelcome = () => {
  const isAppReady = useIsAppReady();
  const allWorkspaces = hooks.useAllWorkspaces();
  const wasIntroOpened = hooks.useWasIntroOpened();
  const openIntro = hooks.useOpenIntroAction();

  useEffect(() => {
    if (isAppReady && !wasIntroOpened && isEmpty(allWorkspaces)) openIntro();
  }, [isAppReady, allWorkspaces, wasIntroOpened, openIntro]);
};
```

Comment (konekoya) [reply ohara-manager/client/src/store/reducers/ui/intro.js]: Thanks for the explanation :)

**Pull request merged**